kod/object/active/holder/room.kod

```diff
@@ -963,6 +963,11 @@ messages:
       {
          return RID_BRAX;
       }
+      
+      if (piRoom_num >= RID_DESERT) and (piRoom_num <= RID_DESERT_END)
```

Comment:
See the comment attached to this message - nothing else should be added here. Rooms need to override this message with their own region (and is why I added the GetRegion() override to desert.kod).